metaci/build/tasks.py

```diff
@@ -93,14 +94,23 @@ def run_build(build_id, lock_id=None):
 
 
 def start_build(build, lock_id=None):
-    queue = django_rq.get_queue(build.plan.queue)
-    result = queue.enqueue(
-        run_build, build.id, lock_id, job_timeout=build.plan.build_timeout
-    )
-    build.task_id_check = None
-    build.task_id_run = result.id
-    build.save()
-    autoscale()
+    queue_name = build.plan.queue
+    if queue_name == "perftest":
+
+        class Result(T.NamedTuple):
+            id: T.Any
```

Comment:
Do we need to return anything? And if so why a namedtuple?

Reply:
We need to return an object with an "id" property because upstream code is expecting to log the "result.id" which I guess is the Django job id.